From 3ddc68db1472d5c164d319debe031f42f8846ba1 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 22 May 2022 03:25:06 +0200
Subject: [PATCH 329/406] media: i2c: imx258: Add i2c supply

Sensor depends on I2C bus to be powered up to work.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/i2c/imx258.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/imx258.c b/drivers/media/i2c/imx258.c
index e63ec7b4df0c..486d685f34ce 100644
--- a/drivers/media/i2c/imx258.c
+++ b/drivers/media/i2c/imx258.c
@@ -607,6 +607,7 @@ static const char * const imx258_supply_names[] = {
 	"vana", /* Analog (2.8V) supply */
 	"vdig", /* Digital Core (1.5V) supply */
 	"vif",  /* Digital I/O (1.8V) supply */
+	"i2c",  /* I2C BUS I/O (1.8V) supply */
 };
 
 #define IMX258_SUPPLY_COUNT ARRAY_SIZE(imx258_supply_names)
-- 
2.34.1

